modules/notification/webhook/webhook.go

```diff
@@ -398,3 +398,50 @@ func (m *webhookNotifier) NotifyDeleteComment(doer *models.User, comment *models
 		go models.HookQueue.Add(comment.Issue.Repo.ID)
 	}
 }
+
+func (m *webhookNotifier) NotifyIssueChangeLabels(doer *models.User, issue *models.Issue,
```

Comment:
I see that `ClearLabels()` doesn't use this function. Is it a whole type of notification?